### PR TITLE
Update fields in Equipment Experts API

### PR DESCRIPTION
**Pull request by Shinsina**
The CSV is only there because it constantly ends back up in my staging area even if I discard local changes, delete the file, etc.

### CODE REVIEW

File: packages/equipment-experts/index.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ module.exports = ({
       next: page < lastPage ? linkTo(req, page + 1, limit) : '',
     },
     meta: {
+      path: 'https://github.com/Shinsina/randall-reilly-websites',
```
Comment (solocommand) [inline packages/equipment-experts/index.js:64]: The CSV is fine, it's fixing whitespace that should have been fixed previously. However, this isn't the correct value for the `meta.path` parameter. Check the old API (in the trello card) to see what was present for this field

**Review by solocommand — CHANGES_REQUESTED** [review]

**Comment by solocommand** [comment]
[Modify Equipment Experts API](https://trello.com/c/RrNgpEmG/63-modify-equipment-experts-api)

File: packages/equipment-experts/index.js (hunk 1 above)
Comment (solocommand) [reply packages/equipment-experts/index.js:64]: Most likely you're going to want to use the request object to the the current URL. IIRC this is what they used to key off of for pagination in their old system.

Comment (Shinsina) [reply packages/equipment-experts/index.js]: My bad on the URL I had to reset my local instance of the repo and that just so happened to be what was on my clipboard.

**Comment by Shinsina** [comment]
I figured out what your comment meant give me a bit

**Review by solocommand — APPROVED** [review]

**Pull request merged**